Library/Formula/fpc.rb

```diff
@@ -28,11 +28,15 @@ def install
 
     # Prevent non-executable audit warning
     rm_f Dir[bin/"*.rsj"]
+
+    # Generate a default fpc.cfg to set up unit search paths
+    system "#{bin}/fpcmkcfg", "-p", "-d", "basepath=#{lib}/fpc/#{version}", "-o", "#{etc}/fpc.cfg"
   end
 
   test do
     hello = <<-EOS.undent
       program Hello;
+      uses GL;
```

Comment:
That is my understanding. According to the following snipped (from the url above) users have at least two additional paths for a customized cfg file, before the default one is loaded. The binary installer also does something like this, installs a fpc.cfg to ../etc (wrt the fpc binary) and then lets users create a custom cfg for their needs.

```
Unless you specify the -n (see page 96) option, the compiler will look for a configuration file fpc.cfg in the following places:
    Under unix (such as linux)
        The current directory.
        Your home directory, it looks for .fpc.cfg.
        The directory specified in the environment variable PPC_CONFIG_PATH, and if it is not set, it will look in the etc directory above the compiler directory. (For instance, if the compiler is in /usr/local/bin, it will look in /usr/local/etc)
        The directory /etc.
```